### PR TITLE
Add Feb 2022 SFB Galaxy workshop event

### PR DESCRIPTION
**Pull request by pavanvidem**

### CODE REVIEW

**Comment by pavanvidem** [comment]
does it make sense to have crc992, crc1425 and meinbio in supporters?

**Comment by beatrizserrano** [comment]
> does it make sense to have crc992, crc1425 and meinbio in supporters?

to me, yes, but I think that meinbio is missing from the list of [sponsors](https://github.com/usegalaxy-eu/website/blob/master/_includes/sponsors.html). You need to add it if you want the logo to appear at the bottom.

Thanks @pavanvidem!!

File: _events/2022-02-21-GalaxyWS_SFB.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+We are offering a Galaxy workshop on high-throughput data analysis. This is an online course and is limited to the members CRC 1425, CRC 992 and MeInBio.
+
+## Venue
+
+Online
+
+
+# Important notes
```
Comment (beatrizserrano) [inline _events/2022-02-21-GalaxyWS_SFB.md:28]: shouldn't these go in the preparation section?

**Review by beatrizserrano — APPROVED** [review]

**Comment by bgruening** [comment]
The date in the filename needs to be changed to be NOT in the future.

**Pull request merged**